packages/hardhat-plugin-viem/src/modules.ts

```diff
@@ -0,0 +1 @@
+export { buildModule } from "@nomicfoundation/ignition-core";
```

Comment:
Let's not export this here. Users will import from `hardhat-ignition`

Reply:
@fvictorio wdty? this is a similar dynamic to the toolbox and the network helpers. With the difference that we have two different plugins: ethers, and viem. My worry is that if we reexport we'd partition the userbase & docs into two

Reply:
I'm not sure I understand that are the options here

Reply:
We have opted for option 2.